internal/cli/status.go

```diff
@@ -0,0 +1,992 @@
+						rr.Platform,
+						rr.CategoryDisplayHint.String(),
+					}
+
+					// Add column data to table
+					releaseResourcesTbl.Rich(
+						columns,
+						[]string{
+							statusColor,
+						},
+					)
+				}
+			}
+
+		}
+	} // else show no table
+
+	appHeaders := []string{
+		"App", "Workspace", "Deployment Status", "Deployment Checked", "Release Status", "Release Checked",
+	}
+
+	appTbl := terminal.NewTable(appHeaders...)
+
+	statusColor := ""
+	columns := []string{
+		app.Name,
+		workspace,
+		deployStatusReportComplete,
+		deployStatusReportCheckTime,
+		releaseStatusReportComplete,
+		releaseStatusReportCheckTime,
+	}
+
+	// Add column data to table
+	appTbl.Rich(
+		columns,
+		[]string{
+			statusColor,
+		},
+	)
+
+	// TODO(briancain): we don't yet store a list of recent events per app
+	// but it would go here if we did.
+	// Recent Events
+	//   Events List
+
+	if c.flagJson {
+		c.outputJsonAppStatus(appTbl, deployTbl, resourcesTbl, releaseTbl, releaseResourcesTbl, project)
+	} else {
+		c.ui.Output("")
+		c.ui.Output("Application Summary")
+		c.ui.Table(appTbl, terminal.WithStyle("Simple"))
+		c.ui.Output("")
+		c.ui.Output("Deployment Summary")
+		c.ui.Table(deployTbl, terminal.WithStyle("Simple"))
+		c.ui.Output("")
+		c.ui.Output("Deployment Resources Summary")
+		c.ui.Table(resourcesTbl, terminal.WithStyle("Simple"))
+		c.ui.Output("")
+
+		if !releaseUnimplemented {
+			c.ui.Output("Release Summary")
+			c.ui.Table(releaseTbl, terminal.WithStyle("Simple"))
+			c.ui.Output("")
+			c.ui.Output("Release Resources Summary")
+			c.ui.Table(releaseResourcesTbl, terminal.WithStyle("Simple"))
+			c.ui.Output("")
+		}
+
+		c.ui.Output(wpStatusAppSuccessMsg)
+	}
+
+	if appFailures {
+		c.ui.Output("")
+
+		c.ui.Output(wpStatusHealthTriageMsg, projectTarget, terminal.WithWarningStyle())
+	}
+
+	return nil
+}
+
+// FormatProjectStatus formats all known projects into a table
+func (c *StatusCommand) FormatProjectStatus() error {
+	if !c.flagJson {
+		c.ui.Output(wpStatusMsg, c.serverCtx.Server.Address)
+	}
+
+	// Get our API client
+	client := c.project.Client()
+
+	projectResp, err := client.ListProjects(c.Ctx, &empty.Empty{})
+	if err != nil {
+		c.ui.Output("Failed to retrieve all projects:"+clierrors.Humanize(err), terminal.WithErrorStyle())
+		return err
+	}
+	projNameList := projectResp.Projects
+
+	headers := []string{
+		"Project", "Workspace", "App Statuses",
+	}
+
+	tbl := terminal.NewTable(headers...)
+
+	for _, projectRef := range projNameList {
+		resp, err := client.GetProject(c.Ctx, &pb.GetProjectRequest{
+			Project: projectRef,
+		})
+		if err != nil {
+			return err
+		}
+
+		workspace, err := c.getWorkspaceFromProject(resp)
+		if err != nil {
+			return err
+		}
+
+		// Get App Statuses
+		var appStatusReports []*pb.StatusReport
+		var ready, alive, down, unknown int
+		for _, app := range resp.Project.Applications {
+			appStatusResp, err := client.GetLatestStatusReport(c.Ctx, &pb.GetLatestStatusReportRequest{
```

Comment:
I think I'd rather just show both deployment and release statuses as two separate columns in the project view. Then we don't have to guess and people can interpret them separately rather than trying to combine deployment and releases statuses as 1 health report. This will also match the rest of the views too.